cypress/e2e/tests/migration/applicationinventory/assessment/import.test.ts

```diff
@@ -252,6 +252,29 @@ describe(["@tier2"], "Application import operations", () => {
         verifyImportErrorMsg(errorMsgs);
     });
 
+    it("Import .CSV file with missing application name", function () {
+        // Automates Polarion MTA-368
+        Application.open();
+        cy.wait("@getApplication");
+
+        // Import csv with invalid record type
+        const fileName = "missing_application_name.csv";
+        importApplication(filePath + fileName);
+        cy.wait(2000);
+
+        // Open application imports page
+        openManageImportsPage();
+
+        // Verify import applications page shows correct information
+        verifyAppImport(fileName, "Completed", 0, 1);
+
+        var errorMsgs = ["Application Name is mandatory."];
```

Comment:
Please use `const` in this case.

Apart from that, if you're going to verify just one error it can be a string, the `verifyImportErrorMsg` checks if the parameter is an array or a string.

![image](https://github.com/konveyor/tackle-ui-tests/assets/117646518/5639402b-03d0-45c7-ac36-cd8a9d61af3c)

Reply:
In a few places, an array is passed to  verifyImportErrorMsg(), so retaining verifyImportErrorMsg() as is .